tests/utils/renderInThemeAndStoreProviders.js

```diff
@@ -21,4 +21,20 @@ const AllProviders = ({ children }) => {
   );
 };
 
+jest.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str) => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {}),
+      },
+    };
+  },
+  initReactI18next: {
+    type: '3rdParty',
+    init: () => {},
+  },
+}));
+
```

Comment:
Could you add a comment to provide the source of this code please? (https://react.i18next.com/misc/testing)

Reply:
My bad. This code is no longer useful. It was for handling trad but we have found a workaround.